src/lib/queryAdvise.test.js

```diff
@@ -0,0 +1,11 @@
+import { getData } from './queryAdvise';
+import { renderIntoDocument } from '../../tests/utils/dom-render';
+
+describe('queryAdvise', () => {
```

Comment:
No need for this `describe`, since the filename's already saying we're testing `queryAdvise`